src/execution/execute.ts

```diff
@@ -261,6 +265,7 @@ export interface ExecutionArgs {
   fieldResolver?: Maybe<GraphQLFieldResolver<any, any>>;
   typeResolver?: Maybe<GraphQLTypeResolver<any, any>>;
   subscribeFieldResolver?: Maybe<GraphQLFieldResolver<any, any>>;
+  signal?: AbortSignal;
```

Comment:
I think it would be reasonable to add this to the top-level `GraphQLArgs`/`graphql` function as well.

Reply:
Not quite sure I understand the idea, can you explain why?

The way I understand it: clients (graphql servers) are calling `execute/experimentalExecuteIncrementally` and that's exactly the place where they will be passing an abort signal.

Reply:
There's a function called `graphql` that you can use that wraps a bunch of stuff (parsing, validation, execution) into a single function call. So it might be reasonable for users of that API to be able to set a signal (it's just a matter of adding it to GraphQLArgs and threading it through to execute).

Reply:
Thanks. I added signal to the GraphQLArgs/graphql call